examples/ReferenceTestV2/reference_consumer_v2.py

```diff
@@ -67,19 +67,16 @@ def __init__(self):
     def log_result(self, is_ok: bool | None, step: str, info: str, extra_info: str | None = None):
         xtra = f' ({extra_info}) ' if extra_info else ''
         self._results.append(ResultEntry(is_ok, step, info, xtra))
-        #
-        # if is_ok is None:
-        #     self._results.append(f'{step} => no result {xtra}{info}')
-        # elif is_ok:
-        #     self._results.append(f'{step} => passed {xtra}{info}')
-        # else:
-        #     self._results.append(f'{step} => failed {xtra}{info}')
 
     def print_summary(self):
         print('\n### Summary ###')
         for r in self._results:
             print(r)
 
+    @property
+    def failed_count(self):
+        return len([r for r in self._results if r.verdict is False])

```

Comment:
can be optimized to `sum(1 for r in self._results if r.verdict is False)`

Reply:
no, that is just different.